src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -202,4 +203,12 @@ public static Set<MedicalHistory> parseMedicals(Collection<String> medHistories)
         }
         return historiesSet;
     }
+
+    /**
+     * Returns true if at least 1 of the prefixes isn't the empty {@code Optional} values in the given
+     * {@code ArgumentMultimap}.
+     */
+    public static boolean isAnyPrefixPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
+        return Stream.of(prefixes).anyMatch(prefix -> argumentMultimap.getValue(prefix).isPresent());
+    }
```

Comment:
Nice work refactoring this! Would it be better to include directly as a method in `argumentMultimap` though?